README.md

```diff
@@ -1,70 +1,155 @@
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-### `npm run build`
+<!-- PROJECT DESCRIPTION -->
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+# Live app<a name="live-link"></a>
+** https://paulshaun-kdtk.github.io/capstone-Car-Show/
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+# Video explanation of app <a name="video-explanation"> </a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+** https://www.loom.com/share/20f584469444424c98a00431bf840a8b
 
-### `npm run eject`
+# 📖 React Capstone TV Shows Metrics <a name="about-project"></a>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+**React Capstone TV Shows Metrics:** is a simple advertisement site, configured with linters for html, css  and javascript
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 🛠 Built With <a name="built-with"></a>
+ HTMl
+ CSS
+ Bootstrap
+ Javascript
+### Tech Stack <a name="tech-stack"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">React</a></li>
+     <li><a href="#">javascript</a></li>
+     <li><a href="#">Axios</a></li>
+     <li><a href="#">TV Shows Api</a></li>
+  </ul>
+</details>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
 
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+<!-- Features -->
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+### Link to Loom technical presentation
+<a href="https://www.loom.com/share/a863515a0867407a8c6324ce43d68ba2?sid=2fb8d742-a76f-4d44-98ac-8a01399f7b6f"> Lomm technical presentaion </a>
 
```

Comment:
- [ ] Great job on your project presentation! 👏 However, it appears that the video presentation link provided doesn't match the content for this particular project. To ensure clarity and accuracy, consider updating the link with the correct video presentation.

   - Don't worry, it's a common oversight, and we're here to make sure everything aligns perfectly. If you need any assistance with this or have any questions, feel free to reach out. 
   
Happy coding! 😊🚀